kubectl-fdb/cmd/deprecation.go

```diff
@@ -193,3 +193,29 @@ func checkDeprecation(cmd *cobra.Command, kubeClient client.Client, inputCluster
 	cmd.Printf("%d cluster(s) without deprecations\n", clusterCounter)
 	return nil
 }
+
+func getMinimalYAML(object interface{}) ([]byte, error) {
```

Comment:
That could also be a `FoundationDBClusterSpec`? I assume you used an `interface{}` to use that method later for different structs?

Reply:
Yes, that was my thinking.